### PR TITLE
Implementation of home.html and about.html

### PR DESCRIPTION
**Pull request by Richie9000**
Create the 2 pages for mobile.
Adapt the 2 pages to desktop.
Implement the section "Featured speakers" with dynamic HTML.
This is the link for the video documentation:
https://www.loom.com/share/14d36b91364a4390b0dcbd407b8bf6f1

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    <script src="script.js" defer></script>
+    <title>Estudio Polanco</title>
+  </head>
+  <body>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn">&times;</a>
+      <div class="overlay-content">
+        <a href="#work" id="mybtn">Program</a>
+        <a href="#about">About</a>
+        <a href="#contact">Gallery</a>
```
Comment (sinansevgi) [inline about.html]: - [ ] I should be able to navigate between the home page and the about page. Please update your menu links accordingly.

File: home.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
```
Comment (sinansevgi) [inline home.html:1]: - [ ] Both of your pages have horizontal scrolls. It looks bad and breaks the user experience. You should fix this.
Screenshot 2021-07-03 at 01-21-53 Polanco Studio

File: home.html [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+              class="topbar-icon"
+              src="./img/Twitter.svg"
+              alt="Twitter logo"
+            />
+          </a>
+        </li>
+        <li>Logout</li>
+        <li>Mypage</li>
+        <li>Korean</li>
+      </ul>
```
Comment (sinansevgi) [inline home.html]: - [ ] Menu items not aligned correctly, some of them overflowing and not visible. You should fix the alignment issue and give them proper margins.
Screenshot 2021-07-03 at 00-55-36 Polanco Studio

File: home.html [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+          <a href="#">About</a>
+        </li>
+        <li>
+          <a href="#">Program</a>
+        </li>
+        <li>
+          <a href="#">Gallery</a>
+        </li>
+      </ul>
+    </nav>
```
Comment (sinansevgi) [inline home.html]: - [ ] The menu section isn't aligned correctly. You should be sure that items aligned as in the design template.
- [ ] The menu doesn't have correct styling (color, font, etc..) You should be sure that your project looks exactly like the design template.

Please check out the screenshots to compare your implementation and the template

The Template :arrow_down: 
Screenshot from 2021-07-03 01-28-02

The Project :arrow_down: 
Screenshot 2021-07-03 at 00-55-47 Polanco Studio

File: home.html [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+      </header>
+      <main>
+        <div class="welcome">
+          <h2>"Hello! sharing world"</h2>
+          <h1>
+            ESTUDIO POLANCO GLOBAL SUMMIT <br />
+            2021
+          </h1>
+        </div>
+        <div class="paragraph">
```
Comment (sinansevgi) [inline home.html:83]: - [ ] You should use the correct background color for this section. For this project we expect you to create a page that exactly looks like the design template. Please be sure that you use the color pallette given in the template.

Screenshot 2021-07-03 at 00-56-08 Polanco Studio

File: home.html [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+            Polanco.
+          </p>
+        </div>
+        <div class="paragraph-two">
+          <h3>2021.11.11(THU) ~ 16 (FRI)</h3>
+          <p>@ estudio.polanco.ciudad de Mexico.</p>
+          <p></p>
+        </div>
+      </main>
+      <section class="program">
```
Comment (sinansevgi) [inline home.html]: - [ ] Items in this container should have less margin between them.
- [ ] Items in this container should have less bottom padding.
- [ ] Titles are overflowing the images. You should be sure it aligned as in the design template.
- [ ] Section should be smaller on desktop resolution. Currently, It is too big.

Please compare the screenshot of the template and the project

The template :arrow_down: 
Screenshot 2021-07-03 at 01-41-23 CC Global Summit 2015

The Project :arrow_down: 
Screenshot from 2021-07-03 01-29-27

File: home.html [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+      </section>
+      <div class="button-more">
+        <button type="button">MORE &#5167;</button>
+      </div>
+      <div class="desktop-logo">
+        <div class="title-about-four">
+          <h5>Partners</h5>
+          <hr />
+        </div>
+        <div class="logos">
```
Comment (sinansevgi) [inline home.html:185]: - [ ] This section is not aligned correctly and items have too much margin between them. You should place them correctly and reduce the margin between them. 

The Project :arrow_down: 
Screenshot from 2021-07-03 00-58-01

The Design Template :arrow_down: 
Screenshot 2021-07-03 at 02-01-42 CC Global Summit 2015

File: script.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+  pName.className = 'name-model';
+  divInfo.appendChild(pName);
+  const pData = document.createElement('p');
+  pData.innerHTML = models[i].data;
+  pData.className = 'data-model';
+  divInfo.appendChild(pData);
+  const pBio = document.createElement('p');
+  pBio.innerHTML = models[i].biography;
+  pBio.className = 'paragraph-model';
+  divInfo.appendChild(pBio);
```
Comment (sinansevgi) [inline script.js]: - [ ] You should introduce a function for creating the DOM elements. This way your code will be more organized and maintainable.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,918 @@
...
+}
+
+.backimg-about {
+  background: url(./img/about-desktop-back.png) no-repeat center center fixed;
+  background-size: cover;
+  height: 100vh;
+  margin-left: -7vw;
+  margin-right: -7vw;
+  padding-bottom: 11vh;
+  margin-top: -3.5vh;
```
Comment (sinansevgi) [inline style.css:22]: [Optional]
- [ ] Negative margins have incorrect behavior on mobile devices and affect elements that occur after them in the document flow. You should avoid using them.

File: home.html [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+            src="./img/Google_Capital_logo.svg"
+            alt="Logo"
+            class="img-logos"
+          />
+          <img src="./img/amazon.svg" alt="Logo" class="img-logos" />
+          <img src="./img/Gsuite_logo.svg" alt="Logo" class="img-logos" />
+          <img src="./img/Pixel_wordmark.svg" alt="Logo" class="img-logos" />
+        </div>
+      </div>
+      <footer>
```
Comment (sinansevgi) [inline home.html:197]: - [ ] You should place the logo and the copyright text as same as given in the design template. Currently, Items not placed and aligned correctly.

Screenshot :arrow_down_small: 
Screenshot from 2021-07-03 00-58-14

File: style.css [hunk 11]
```diff
@@ -0,0 +1,918 @@
...
+  justify-content: center;
+  height: 10vh;
+}
+
+.img-models {
+  width: 22vw;
+  border-radius: 2px;
+  height: 15vh;
+  margin: 3vh;
+}
```
Comment (sinansevgi) [inline style.css]: - [ ] Images should protect their ratio in mobile and desktop resolutions. In your implementation, Images stretch in mobile and higher resolutions. Please be sure images behave as seen in the design template.

Mobile
Screenshot 2021-07-03 at 01-31-48 Polanco Studio
Desktop
Screenshot 2021-07-03 at 01-32-04 Polanco Studio

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @Richie9000 , 

Good job so far! Great work with the about page it looks really good.

There are some issues that you still need to work on to go further but you are almost there!

**Please be sure the page looks almost identical to the original design, outside the content differences.
The pages should use the exact same colors, fonts, backgrounds, borders, etc, from the original design**



 ## Required Changes ♻️

- [ ]  **You should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes**.  Please provide the video link in the pull request description.
The content of the video should include:
   - A description of the project.
   - A demo of the project features (different pages and different screen sizes).
   - You should also highlight some interesting piece of code or something you built that you are very proud of.
- [ ] You should have a proper readme file. Please update the readme template accordingly.
- [ ] The README file should contain an "Acknowledgements" section that gives appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License.
- [ ] You should deploy the project to Github Pages.
- [ ] Your readme file should present a link to deployment (Live Demo Link)

_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+      rel="stylesheet"
+    />
+    <script src="script.js" defer></script>
+    <title>Estudio Polanco</title>
+  </head>
+  <body>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn">&times;</a>
+      <div class="overlay-content">
+        <a href="#work" id="mybtn">Program</a>
```
Comment (MahmoudBakr23) [inline about.html]: - [ ] Kindly fix the navigation to the home page from the about page!

File: about.html [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+          </a>
+        </li>
+        <li>Logout</li>
+        <li>Mypage</li>
+        <li>Korean</li>
+      </ul>
+    </div>
+    <nav>
+      <div class="image-nav">
+        <img src="./img/logo-footer.png" alt="Image Logo" />
```
Comment (MahmoudBakr23) [inline about.html]: - [ ] **OPTIONAL**
Please make the logo navigates to the home page with a link `<a>`.

_Please notice that this a high-recommended to fix requirement, and I'm sorry it didn't show up earlier 🙏 😄_

File: home.html (hunk 2 above)
Comment (MahmoudBakr23) [reply home.html:1]: Fixed!

File: home.html (hunk 3 above)
Comment (MahmoudBakr23) [reply home.html]: Fixed!

File: home.html [hunk 14]
```diff
@@ -0,0 +1,207 @@
...
+      </header>
+      <main>
+        <div class="welcome">
+          <h2>"Hello! sharing world"</h2>
+          <h1>
+            ESTUDIO POLANCO GLOBAL SUMMIT <br />
+            2021
+          </h1>
+        </div>
+        <div class="paragraph">
```
Comment (MahmoudBakr23) [inline home.html:83]: - [ ] As required, please stick to these colors of the given design.
* Basic color swatch: #272A31, #EC5242, #D3D3D3

File: home.html [hunk 15]
```diff
@@ -0,0 +1,207 @@
...
+      <div class="button-more">
+        <button type="button">MORE &#5167;</button>
+      </div>
+      <div class="desktop-logo">
+        <div class="title-about-four">
+          <h5>Partners</h5>
+          <hr />
+        </div>
+        <div class="logos">
+          <img src="./img/Apple_logo.svg" alt="Logo" class="img-logos" />
```
Comment (MahmoudBakr23) [inline home.html:186]: - [ ] As required, please use the correct background color and spacings regarding this section!
- [ ] Good job in doing some of the changes on the programs section. However, and as required, kindly do the rest of the corresponding changes to match the given design.

File: home.html [hunk 16]
```diff
@@ -0,0 +1,207 @@
...
+            src="./img/Google_Capital_logo.svg"
+            alt="Logo"
+            class="img-logos"
+          />
+          <img src="./img/amazon.svg" alt="Logo" class="img-logos" />
+          <img src="./img/Gsuite_logo.svg" alt="Logo" class="img-logos" />
+          <img src="./img/Pixel_wordmark.svg" alt="Logo" class="img-logos" />
+        </div>
+      </div>
+      <footer>
```
Comment (MahmoudBakr23) [inline home.html:197]: - [ ] As required, kindly fix the footer's elements to match the given design.
- [ ] As required, please fix the featured speakers' section's images.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi, @Richie9000😃, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

- [ ] As required, please record a short video to present this project.
- [ ] As required, please check the live demo link on the README file and fix it.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
_Check the **OPTIONAL** comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 12 above)
Comment (Temmarie) [reply about.html]: - [x] As required by the previous TSE, kindly fix the navigation from the about page to the home page on mobile screens.
- The link to the home page doesn't show in the dropdown menu. 
- The menu items don't work, so navigation on mobile screens is impossible.
- The menu doesn't close after clicking on a menu item. Kindly fix this.

- [ ] Also, please follow the given design template, the fonts, spacings and colours.

File: home.html (hunk 3 above)
Comment (Temmarie) [reply home.html]: - [ ] It seems while making some of the previously required changes, this issue came up again. 
- Your webpage on desktop screens are distorted and have a horizontal overflow as well. This is probably due to wrong use of margins. Kindly fix it.

_See images below._
image

File: home.html (hunk 15 above)
Comment (Temmarie) [reply home.html:186]: - [ ] Kindly use the required colour for the footer as requested by the previous reviewers.

_See images below._

**Original** | **Your project**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @Richie9000     ! :wave: :wave:

#### You have done really well in implementing some of the previously required changes.
 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
- [ ] Kindly refer to the [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template and make sure to follow the design as is written in the project's requirements.

- [ ] Also, make all the required changes before submitting for review. 

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
__Kindly make the necessary changes and submit another code review request.__

File: about.html (hunk 12 above)
Comment (Davidosky007) [reply about.html]: * @Richie9000 the issue here pointed out by the previous TSE has still not been addressed kindly make the necessary changes  🙏

File: home.html (hunk 14 above)
Comment (Davidosky007) [reply home.html:83]: fixed 👍

File: home.html (hunk 15 above)
Comment (Davidosky007) [reply home.html:186]: * nice work implementing some of the requested changes, but the background colour is still different from the original( color: #272A31) please use the required colour for the footer

**Comment by Davidosky007** [comment]
Hi, @Richie9000, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
CR3_gif_almost_there


 ## Required Changes ♻️
* The mobile menu is still not working as expected( navigation is not possible and when links are clicked menu dropdown should not display)
* The right colour should be used for the footer section please check comments and make necessary changes 👍 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi, @Richie9000, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
CR3_gif_almost_there

 ## Required Changes ♻️
* The mobile dropdown menu is still not working as the previous reviewer pointed kindly fix it 🙏 
* Footer section still lacks the right colour as stated by the previous reviewer, check comments below and make the necessary changes 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Richie9000 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._
- [ ] As stated in the previous reviews, the dropdown list in your hamburger button of the mobile view lacks proper reference to the different navigated pages. It is not functional for now. Please see the attached gif example on how it can be improved. 
chrome-capture (1)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [ ] Contents are overflowing out of the screen width. Please kindly fix this whenever you can. Though it is optional, it will make your project more complete and professional. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by henatan99 — APPROVED** [review]
Hi @Richie9000 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

- [ ] OPTIONAL: Well done implementing the requested changes. I think you can do more to your project in your spare time, though it is not required for submission. I made a suggestion in my previous review to mind overflowing content out of the screen width and to add one more, there are some elements lacking proper margin/padding. Please see the attached screenshot.

image


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Richie9000** [comment]
Thank you guys!!

**Pull request merged**